tests/ecosystem/upgrade/test_resources.py

```diff
@@ -36,7 +36,7 @@ def test_storage_pods_running(multiregion_mirror_setup_session):
 )
 @pre_upgrade
 @ignore_leftovers
-def test_start_pre_upgrade_pod_io(pre_upgrade_pods_running_io):
+def test_start_pre_upgrade_pod_io(pre_upgrade_pods_running_io, pause_cluster_load):
```

Comment:
```suggestion
def test_start_pre_upgrade_pod_io(pause_cluster_load, pre_upgrade_pods_running_io):
```
I suggest to first pause the bg io and start upgrade io after that. Pytest order for executing fixtures seems to be by [fixture position in test function arguments](https://stackoverflow.com/a/64471427).

Reply:
right

Reply:
done